docs/Using Fleet/manage-access.md

```diff
@@ -50,7 +50,7 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | Run queries designated "**observer can run**" as live queries against all hosts                                                            | ✅       | ✅         | ✅         | ✅    |         |
 | Run any query as [live query](https://fleetdm.com/docs/using-fleet/fleet-ui#run-a-query) against all hosts                                 |          | ✅         | ✅         | ✅    |         |
 | Create, edit, and delete queries                                                                                                           |          |            | ✅         | ✅    | ✅      |
-| View all queries and their reports\**                                                                                                      | ✅       | ✅         | ✅         | ✅    |         |
+| View all queries and their reports                                                                                                      | ✅       | ✅         | ✅         | ✅    |         |
```

Comment:
@rachaelshaw @noahtalerman TLDR; Observers should see all queries, not just queries that have ObserverCanRun enabled.